app/cdap/components/GridTable/components/NestedMenu/index.tsx

```diff
@@ -22,7 +22,7 @@ import { IMenuItem } from 'components/GridTable/components/MenuItemComponent/typ
 import { useNestedMenuStyles } from 'components/GridTable/components/NestedMenu/styles';
 import { INestedMenuProps } from 'components/GridTable/components/NestedMenu/types';
 
-export default function({
+const NestedMenu = ({
```

Comment:
why changed to const and export deafult again
can we simply put export default function plz, CLIENT ASK

Reply:
Done